docs/api/openapi3.json

```diff
@@ -689,6 +689,17 @@
         "description": "ListAccessKeys",
         "operationId": "ListAccessKeys",
         "parameters": [
+          {
+            "in": "header",
+            "name": "Infra-Version",
+            "required": true,
+            "schema": {
+              "description": "Version of the API being requested",
+              "example": "0.13.1+dev",
```

Comment:
The examples should probably not include the `+dev`. Does `+dev` get get removed before publish?

Reply:
it gets generated on release with the release version, so the published one won't have dev in it